pkg/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -601,3 +610,27 @@ func (q *DBJobQueue) jobDependencies(ctx context.Context, conn *pgxpool.Conn, id
 
 	return dependencies, nil
 }
+
+func (q *DBJobQueue) jobDependents(ctx context.Context, conn *pgxpool.Conn, id uuid.UUID) ([]uuid.UUID, error) {
+	rows, err := conn.Query(ctx, sqlQueryDependents, id)
```

Comment:
Hm wait, do you mean put the entire JobStatus (which are 3 separate queries), in 1 tx?

Reply:
So I think it doesn't make sense to query the dependencies in the tx because you can't change those after the fact.

For the dependents however, which are also queried in jobstatus, I'm not sure. Adding a dependency to a job doesn't change the job in any way, it just updates the dependencies table. So I don't fully understand where the inconsistency would arise.

My understanding is that transactions for multiple reads are useful in case subsequent queries are based on the results of previous queries, in which case you'd really want the data consistency. As the job's uuid is consistent throughout here, and it's the only piece of data required for the dependencies/dependents queries, I don't think inconsistency is possible.

Reply:
I was mostly concerned about a write in-between two reads but we don't actually care about this in this case since both possible outcomes are valid.